### ISSUE TITLE
"no net charge"  should be changed to "zero net charge" everywhere

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#303]
From discussion in design doc, "no net charge" should be changed to "zero  net charge" to avoid repetition of "no" in cases like "Balloon has no net charge, showing no charges."

**Comment by jessegreenberg** [comment]
@terracoda done in master. Closing.
